Source/towners.cpp

```diff
@@ -554,11 +554,11 @@ void ProcessTowners()
 	}
 }
 
-ItemStruct *PlrHasItem(int pnum, int item, int &i)
+ItemStruct *PlrHasItem(int pnum, int item, int *i)

```

Comment:
We could rename to `i_ptr` then do `auto &i = *i_ptr` to reduce the number of merge conflicts

Reply:
The overload approach is better